package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "start": "node build-system/pr-check.js",
     "lint": "eslint .",
-    "fix": "eslint . --fix"
+    "fix": "eslint . --fix",
+    "replace-secrets": "node build-system/replace-secrets.js"
```

Comment:
Cloud Build can't arbitrarily run `node`; it has to use one of a limited selection of Cloud Builders. NPM was the obvious choice, which is why this is added as an NPM script.